src/StatusQ/Controls/StatusImageCrop.qml

```diff
@@ -94,6 +94,17 @@ Item {
     */
     property real wallTransparency: 0.7
 
+    /*!
+        \qmlproperty bool StatusImageCrop::allowZoomToFitAllContent
+        Minimum allowed zoom. This is 1 if
```

Comment:
This is 1 if ...?

Reply:
Copy-pasta leftovers. Fixed,  thanks for the heads up.